doc/design.txt

```diff
@@ -210,15 +210,15 @@ An image trailer has the following structure:
      0                   1                   2                   3
```

Comment:
> Will the other 32 states reliably trigger a "panic" swap type in the new update code?

Nope, a weird state should either be `NONE` or `FAIL` which end up with somewhat similar behavior.